packages/gen_js_api/gen_js_api.1.0.8/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Easy OCaml bindings for JavaScript libraries"
+description: """
+
+gen_js_api aims at simplifying the creation of OCaml bindings for
+JavaScript libraries.  Authors of bindings write OCaml signatures for
+JavaScript libraries and the tool generates the actual binding code
+with a combination of implicit conventions and explicit annotations.
+
+gen_js_api is to be used with the js_of_ocaml compiler.
+ """
+maintainer: ["Alain Frisch <alain.frisch@lexifi.com>"]
+authors: [
+  "Alain Frisch <alain.frisch@lexifi.com>"
+  "Sebastien Briais <sebastien.briais@lexifi.com>"
+]
+license: "MIT"
+homepage: "https://github.com/LexiFi/gen_js_api"
+bug-reports: "https://github.com/LexiFi/gen_js_api/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "ppxlib" {>= "0.22"}
+  "js_of_ocaml-compiler" {with-test}
+  "conf-npm" {with-test}
+  "ojs"
+  "odoc" {with-doc}
+]
+conflicts: [
+  "js_of_ocaml-compiler" {< "3.0.0"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
```

Comment:
```suggestion
]
x-ci-accept-failures: [
  "ubuntu-16.04"
]
```

Reply:
This is just for the CI, you can upstream it if you want, but if the opam file is autogenerated, you can safely ignore it